src/base/linalg/owl_base_linalg_generic.ml

```diff
@@ -7,6 +7,8 @@ type ('a, 'b) t = ('a, 'b) Owl_base_dense_ndarray_generic.t
 
 module M = Owl_base_dense_ndarray_generic
 
+open Owl_exception
```

Comment:
This is used very rarely, I think it is better to name it when needed instead of a toplevel open

Reply:
Thanks for reviewing! I've updated the code accordingly.